cluster/common.sh

```diff
@@ -1167,6 +1240,10 @@ function parse-master-env() {
   KUBELET_KEY_BASE64=$(get-env-val "${master_env}" "KUBELET_KEY")
   MASTER_CERT_BASE64=$(get-env-val "${master_env}" "MASTER_CERT")
   MASTER_KEY_BASE64=$(get-env-val "${master_env}" "MASTER_KEY")
+  AGGR_CA_KEY_BASE64=$(get-env-val "${master_env}" "AGGREGATOR_CA_KEY")
+  AGGR_CA_CERT_BASE64=$(get-env-val "${master_env}" "AGGREGATOR_CA_CERT")
+  AGGR_PROXYCLIENT_CERT_BASE64=$(get-env-val "${master_env}" "PROXY_CLIENT_CERT")
+  AGGR_PROXYCLIENT_KEY_BASE64=$(get-env-val "${master_env}" "PROXY_CLIENT_KEY")
```

Comment:
s/AGGR//
s/PROXYCLIENT/PROXY_CLIENT/

This code is already super confusing, keeping the env var names consistent will at least not make it worse :)